framework-crates/objc2-accessibility/Cargo.toml

```diff
@@ -69,6 +69,13 @@ block2 = ["dep:block2"]
 objc2-core-foundation = ["dep:objc2-core-foundation"]
 objc2-core-graphics = ["dep:objc2-core-graphics"]
 
+unstable-core-ffi-objc = [
+    "objc2-core-foundation?/unstable-core-ffi-objc",
+    "objc2-core-graphics?/unstable-core-ffi-objc",
+    "objc2-foundation/unstable-core-ffi-objc",
+    "objc2/unstable-core-ffi-objc",
```

Comment:
I would like for the Cargo.toml features to remain "unstable" in the sense that I'm free to remove them in a patch release - but by mentioning them across crates like this, that is no longer the case. Could we remove them here, and instead require users to manually enable the unstable features for all the `objc2` crates they depend on?

I know that'd be slightly less convenient for users, but in practice, most users already directly depend on all the `objc2-*` crates in their dependency tree (e.g. if wgpu were to test it, they'd have to enable `"unstable-core-ffi-objc"` in `objc2`, `objc2-core-foundation`, `objc2-foundation`, `objc2-core-foundation`, `objc2-metal` and `objc2-quartz-core`, all of which are direct dependencies in https://github.com/gfx-rs/wgpu/pull/5641).